src/docs/Foundations/Typography/HeadingStyles/previews/typographyHeading-colors.preview.scss

```diff
@@ -0,0 +1,45 @@
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'typography/_t.headings';
+
+.example {
+  padding: $mu200;
+  max-width: magic-unit-rem(38);
+  margin: 0 auto;
+
+  p {
+    margin: $mu100 0;
+  }
+
+  &__title {
```

Comment:
There is no  `example__title` element in the corresponding preview file. Maybe we can remove that selector?